package.json

```diff
@@ -21,10 +21,15 @@
   "scripts": {
     "coverage": "istanbul cover _mocha -x src/model.js --  --compilers js:babel-core/register --recursive -r jsdom-global/register",
     "start": "node example/server.js",
-    "test": "eslint . && mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "test": "mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "lint": "eslint .",
```

Comment:
I think we need to add a .eslintignore w/ the following content:

```
coverage/*
dist/*
```

could you please add that?